public/ace-themes/sql_console.js

```diff
@@ -1,19 +0,0 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-import * as ace from 'brace';
-
-ace.define('ace/theme/sql_console', ['require', 'exports', 'module', 'ace/lib/dom'], function (
```

Comment:
I may have missed in this PR where you were previously specifying use of this custom `ace` theme, but for consistency with OpenSearch Dashboards (particularly in dark mode), you should use the `textmate` theme in any ace/EuiCodeEditor component. See https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4609 for an example of similar cleanup in core.

Reply:
Sure. Thank you. The code editor was been used in `feature/model-registry` branch. It's not been used in current application. Is that means we don't need to provide more scss files by using `textmate` theme?